src/test/components/ProfileSummaryView.test.js

```diff
@@ -41,6 +23,40 @@ it('renders ProfileSummaryView correctly', () => {
     </Provider>,
     { createNodeMock }
   );
+  expect(profileSummary.toJSON()).toMatchSnapshot();
+});
 
+it('renders ProfileSummaryView with fillers when summary is not available', () => {
```

Comment:
I think you can do both in the same test, which would be also closer to what happens in the real app:

1. render the summary
2. check the snapshot
3. dispatch the action
4. check another snapshot

The component should be updated after dispatching the action. See https://github.com/devtools-html/perf.html/blob/82ad6e9448bc7c3123924ed750d485701cfd8dc3/src/test/components/ProfileSharing.test.js#L35-L45 for an example :)

If it isn't updated, you can maybe force an update by calling `profileSummary.update(<JSX>)`. But I think it won't be necessary. https://github.com/devtools-html/perf.html/blob/82ad6e9448bc7c3123924ed750d485701cfd8dc3/src/test/components/ButtonWithPanel.test.js#L26-L38 has an example of `update`.